src/main/resources/data/minebound/recipes/myrial_console.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "IWI",
+    "ICI",
+    "IGI"
+  ],
+  "key": {
+    "I": {
+      "item": "minebound:myrial_steel_ingot"
+    },
+    "W": {
+      "item": "minebound:gilded_diamond_wire"
+    },
+    "C": {
+      "item": "minebound:alien_circuitry"
+    },
+    "G": {
+      "item": "minebound:myrial_glass"
```

Comment:
In our documentation i think we had myrial_glass_block. If not currently all the myrial glass is myrial_glass_block in the glass block build. I think it would be better to just put block.

Reply:
Our documentation does say `myrial_glass_block`. I used `myrial_glass` since that was the name prior to your PR. I'll update that.

Reply:
Thanks

Reply:
I have reverted this change as per the changes to #35